Controller/RoleController.php

```diff
@@ -116,8 +117,15 @@ public function createRoleAction(Request $request)
      */
     public function updateRoleAction(Request $request, $roleId)
```

Comment:
@lolautruche Ok, then we should rename the url and everything else too, to be consistent. And contenttype also uses "update", unless you have renamed it recently :)